specs/fault-proof/stage-one/optimism-portal.md

```diff
@@ -229,13 +272,14 @@ Allows a user to [prove](#proven-withdrawal) a withdrawal transaction.
 
 Checks that a withdrawal transaction can be [finalized](#finalized-withdrawal).
 
+- MUST revert if the withdrawal being finalized has already been finalized.
 - MUST revert if the withdrawal being finalized has not been proven.
+- MUST revert if the withdrawal was proven at a timestamp less than the creation timestamp of the
+  dispute game it was proven against, which would signal an unexpected proving bug.
 - MUST revert if the withdrawal being finalized has been proven less than
   [Proof Maturity Delay](#proof-maturity-delay) seconds ago.
-- MUST revert if the withdrawal being finalized was proven against a game that is not a
-  [Valid Game](./anchor-state-registry.md#valid-game).
-- MUST revert if the withdrawal being finalized has already been finalized.
-- MUST otherwise return `true`.
```

Comment:
Was this removed because it just reverts outright now?

Reply:
It actually never returned true - I think this was a spec bug